src/components/floating/floating.svelte

```diff
@@ -96,7 +96,7 @@
   }
 </script>
 
-<div bind:this={floating} class="leo-floating">
+<div on:mouseenter on:mouseleave bind:this={floating} class="leo-floating">
```

Comment:
I want to expose these so I can control when the tooltip should close

Reply:
Yeah, I feel like we should do this more!